src/segment_plastic.go

```diff
@@ -0,0 +1,245 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+)
+
+type plastic struct {
+	props properties
+	env   environmentInfo
+
+	Moved    int
+	Deleted  int
+	Added    int
+	Modified int
+	Unmerged int
+	Changed  bool
+
+	Behind   bool
+	Selector string
+
+	plasticWorkspaceFolder string // .plastic working folder
+
+	cmCommand string
+}
+
+const (
+	// FullBranchPath displays the full path of a branch
+	FullBranchPath Property = "full_branch_path"
+)
+
+func (g *plastic) enabled() bool {
+	if !g.env.hasCommand(g.getCmCommand()) {
+		return false
+	}
+	wkdir, err := g.env.hasParentFilePath(".plastic")
+	if err != nil {
+		return false
+	}
+	if g.shouldIgnoreRootRepository(wkdir.parentFolder) {
+		return false
+	}
+
+	if wkdir.isDir {
+		g.plasticWorkspaceFolder = wkdir.parentFolder
+		return true
+	}
+
+	return false
+}
+
+func (g *plastic) shouldIgnoreRootRepository(rootDir string) bool {
+	value, ok := g.props[ExcludeFolders]
+	if !ok {
+		return false
+	}
+	excludedFolders := parseStringArray(value)
+	return dirMatchesOneOf(g.env, rootDir, excludedFolders)
+}
+
+func (g *plastic) string() string {
+	statusColorsEnabled := g.props.getBool(StatusColorsEnabled, false)
```

Comment:
This is old and needs to be removed, it's only in git for backwards compatibility. We now color the segment based on the color templates.

Reply:
Yes, I was wondering what it's used for and forgot to remove it. After all, it's not really used anyway. 
Will be removed!